src/cmake/Superbuild.cmake

```diff
@@ -150,9 +150,9 @@ add_custom_target(
 
 # make check
 add_custom_target(check
-  COMMAND ${CMAKE_CTEST_COMMAND} --test-dir ${CMAKE_CURRENT_BINARY_DIR}/libtiledbvectorsearch --output-on-failure
+  COMMAND ${CMAKE_CTEST_COMMAND} -C $<CONFIG> --test-dir ${CMAKE_CURRENT_BINARY_DIR}/libtiledbvectorsearch --output-on-failure
 )
 
 add_custom_target(check-ci
-  COMMAND ${CMAKE_CTEST_COMMAND} --test-dir ${CMAKE_CURRENT_BINARY_DIR}/libtiledbvectorsearch --output-on-failure -E \"unit_slicing|unit_ivf_index\"
+  COMMAND ${CMAKE_CTEST_COMMAND} -C $<CONFIG> --test-dir ${CMAKE_CURRENT_BINARY_DIR}/libtiledbvectorsearch --output-on-failure --extra-verbose -E \"unit_slicing|unit_ivf_index|unit_logging\"
```

Comment:
For some reason `unit_logging` sleep interval is so imprecise that it fails the tests. I don't know how to handle this issue and if it is severe enough to care about:

https://github.com/TileDB-Inc/TileDB-Vector-Search/actions/runs/8064265455/job/22027798473?pr=250